### ISSUE TITLE
Community anchor institutions

### ISSUE THREAD

**Issue by 1ec5** [ZeLonewolf/openstreetmap-americana#435]
The map should have point of interest icons for community anchor institutions. These are mostly public-sector points of interest that define a typical populated place.

Every map has a different selection of community anchor institutions, but some common ones on American maps are city halls, courthouses, police stations, fire stations, hospitals (being added in #387), schools (already tracked in #78), libraries, and post offices. Sometimes community centers, museums, and banks are also included. A rule of thumb is that the name of a community anchor institution is somewhat unimportant on the map, because it’s usually named after the community (or vice versa).

Some maps have dedicated icons for each kind of institution, while others pair the generic POI icon with an abbreviated label. Community anchor institutions appear on many maps that don’t include retail POIs in general, so I think it would be reasonable to introduce them at a lower zoom level than the general POI layer. At these lower zoom levels, it would be unnecessary to label the POI by its `name:en`.

<img width="280" alt="Engels (Huff) Hamilton County 1997" src="https://user-images.githubusercontent.com/1231218/174456035-c90a2298-b515-49dc-bca8-c663ca2078f4.png"> <img width="268" alt="Rand McNally Dallas 1994" src="https://user-images.githubusercontent.com/1231218/174456064-848d920e-b084-4cc9-93a0-8014ad166901.png"> <img width="213" alt="Rand McNally Atlanta 2000" src="https://user-images.githubusercontent.com/1231218/174456153-9f2204e9-1e12-40ed-ad12-28f50e73c872.png">

Among these POI types, post offices will be trickier. We’d want to distinguish U.S. Post Offices from storefronts for delivery services (The UPS Store, FedEx Office, and more obscure services). Even though the USPS may be of diminished importance these days, their rural locations in particular are of principal importance to the communities they serve, more than any other POI. Thanks to the [name suggestion index](https://nsi.guide/index.html?t=operators&k=amenity&v=post_office&tt=usps), either `operator:type=public` or `operator:wikidata=Q668687` could be used to make this distinction. However, I don’t think either key is currently recognized by the OpenMapTiles schema.

**Comment by zekefarwell** [comment]
That list looks good.  I'd add:

- Churches
- Parks
- Airports 

Churches may require a variety of icons for different religions, but they are often quite prominent buildings that serve as landmarks even for those who never set foot inside.

Parks and airports are both already rendered, but I think they make sense to include in this list since they are often important hubs.  Of course they can also be quite small and unimportant (same with cemetaries).  Ideally we'd have some way of only showing larger or more important ones at lower zooms.

**Comment by kennykb** [comment]
The initial problem statement from @1ec5 is a little unclear.  Are community anchor institutions:

1. Especially important PoI's, to be rendered perhaps at lower zoom than others?
2. PoI's that, if there are a lot of them nearby, would render a `place` node more important?
3. PoI's whose names may be safely omitted since they're usually named for the community?

If it's 1. or 2., then:

I'd add suburban and interurban train stations. Not every subway or light rail stop, but an Amtrak station or even a station on a commuter line is usually named for the community (or vice versa!).  

Make sure that colleges/universities are included among 'schools'!

I'm particular interested in "what combination of PoI's and population makes a place a village/town/city?" since `place=*` is a hot mess in the areas where I've been working.  If this related question sheds some light, that's all to the good!

**Comment by ZeLonewolf** [comment]
The list of POIs that should be rendered at lower zooms should also include ones that are appropriate in rural areas as well.  For example, campgrounds.  They're pretty much only in more rural areas so POI collision isn't much a problem.  Campgrounds show up rather prominently on AAA maps, for example.  There are probably other categories appropriate to rural America.

**Comment by bgo-eiu** [comment]
There are various types of community center and/or social facility that I see maps of often. Usually just with generic pinpoints, but I have seen them get included alongside these other types of community institutions. I have not checked how or if they're tagged yet but this includes things like [Community Action Centers](http://gis.baltimorecity.gov/egis/rest/services/311/311_ReferenceLayers/MapServer/58), [Cooling Centers](http://gis.baltimorecity.gov/egis/rest/services/311/311_ReferenceLayers/MapServer/63l), [Warming Centers](http://gis.baltimorecity.gov/egis/rest/services/311/311_ReferenceLayers/MapServer/64), and [Community Violence Intervention Centers](https://www.arcgis.com/home/webmap/viewer.html?webmap=62ea7a73e41940cbb0b247161f88032f&extent=-76.7246,39.2843,-76.545,39.3569). These facilities are often connected to other local institutions in some way, for example, the community violence intervention centers are in part operated by local hospitals and are displayed on a map with them. (Police stations also appear but these institutions are meant to be explicitly separate from the police, this is the sort of thing that is common in American cities with particularly corrupt and dysfunctional police forces but probably not everywhere.)

Some of the above are frequent enough that it might make sense more at an intermediate zoom level. Around the same time schools appear would make sense I think.

As far as bare minimum POIs for a place, the rural places I'm familiar with that have at least something in them usually have a [country store](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcountry_store) and a post office, possibly a fire station. The country store wiki article on OSM is really bad by the way, it needs some investigation and revision.

**Comment by bgo-eiu** [comment]
At a glance, a query for `shop=country_store` which excludes the `brand` tag seems to come close to the definition of a country store. https://overpass-turbo.eu/s/1jrA

Otherwise there's a lot of Tractor Supply chain retail stores which don't really fit the description

**Comment by 1ec5** [comment]
I appreciate the enthusiasm for adding points of interest to the style, but I’d like to refocus the discussion a bit. What I’m proposing above is to render **self-explanatory POIs** at a lower zoom level than the general POI layer. In general, I think we should always pair POI icons with labels, to reduce the pressure to add a distinct icon for every possible kind of POI and turn the map into a kind of rebus. But at these zoom levels, labels would add too much visual clutter, even if technically they wouldn’t collide. Fortunately, for some POIs, you don’t really need the name at all:

> A rule of thumb is that the name of a community anchor institution is somewhat unimportant on the map, because it’s usually named after the community (or vice versa).

To belabor the point, in a stereotypical small town, there’s only going to be one of each “standard” town amenity: one city hall, one post office, one fire station. (Or in a city neighborhood, one postal station, one police precinct, one main community center.) Even if there are two fire stations, a user doesn’t need to know which one is Station 1 and which is Station 2 until they zoom in further. The user would want to distinguish the elementary school and the high school, so in #78 I suggest a distinct icon for each school classification.

> Parks and airports are both already rendered, but I think they make sense to include in this list since they are often important hubs.

> For example, campgrounds. They're pretty much only in more rural areas so POI collision isn't much a problem. Campgrounds show up rather prominently on AAA maps, for example. There are probably other categories appropriate to rural America.

Greenspace and transportation facilities are important, but I think we can deal with them separately (and already have to some extent). These are primarily landuse and only secondarily POIs, requiring different considerations. Maybe I shouldn’t have thrown cemeteries into the mix above, because you could make the same argument for the larger ones.

> Churches may require a variety of icons for different religions, but they are often quite prominent buildings that serve as landmarks even for those who never set foot inside.

Churches aren’t interchangeable, so an icon alone wouldn’t be self-explanatory. I’ve seen community maps that label churches with only the denomination but not the name, for brevity, but this approach doesn’t scale well to a variety of communities across a metropolitan area, let alone North America.

> As far as bare minimum POIs for a place, the rural places I'm familiar with that have at least something in them usually have a [country store](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcountry_store) and a post office, possibly a fire station.

These days, more small towns have a dollar store, but I don’t think it would be quite as self-explanatory, nor as useful for getting a sense of the town’s “center of gravity”. That said, if you know of a map that gives dollar stores or country stores the same prominence as city halls, that would be a very interesting counterpoint. If we do render these POIs without labels, maybe the icon needs to be the brand logo in order to be self-explanatory.

> I'm particular interested in "what combination of PoI's and population makes a place a village/town/city?" since place=* is a hot mess in the areas where I've been working. If this related question sheds some light, that's all to the good!

I think this is a different question than what I’m proposing. Whatever criteria you use to classify a place, the map will reflect that in its treatment of the place label. Tying place classification to a particular style’s editorial selection of community anchor institutions would introduce value judgments to the database and probably make this discussion run in circles.

**Comment by ZeLonewolf** [comment]
AAA certainly thinks that campgrounds are the MOST important POIs at low zoom 😁

![20220618_193900.jpg](https://user-images.githubusercontent.com/3254090/174460230-6ee30bfe-a4a8-447e-bf38-19abc6d784c0.jpg)

**Comment by 1ec5** [comment]
Yes, I think this map proves that they aren’t community anchor institutions but rather something to be depicted even _more_ prominently at low zoom levels. You don’t see many city halls or post offices on that map. 😉

**Comment by bgo-eiu** [comment]
OK I think I had a different working definition of community anchor institutions. Maryland likes to use this term a lot and you can typically expect there to be several per neighborhood, if not on your street, going by what I'm familiar with. (See https://data.imap.maryland.gov/datasets/maryland::maryland-community-anchor-institutions-community-anchor-institutions/explore?location=38.900787%2C-76.701170%2C8.46 for example)

**Comment by 1ec5** [comment]
Yes, there are multiple working definitions of “community anchor institution”. It definitely has a different definition in the context of social services. Another definition includes [economically important employers](https://scag.ca.gov/post/anchor-institutions). However, I couldn’t think of a more apt term for the convention of giving certain kinds of POIs extra prominence while glossing over their individual identities.

**Comment by michaelblyons** [comment]
> However, I couldn’t think of a more apt term for the convention of giving certain kinds of POIs extra prominence while glossing over their individual identities.

"Place-eponymous POIs?" (Totally invented nomenclature)

**Comment by bgo-eiu** [comment]
Dollar Stores aren't common in small towns here either, if anything it correlates very closely with urbanity. I should take a closer look at how these things are distributed in other parts of the country since it's easy to assume certain trends are ubiquitous. Small towns typically don't have town halls and it's pretty common for neighborhoods to have several community centers - I'm more inclined to think there's no such thing as a stereotypical American town than that the east coast is just a bit weird.

**Comment by ZeLonewolf** [comment]
Okay, we are _not_ rendering dollar stores at lower zooms :-P

**Comment by zekefarwell** [comment]
> The country store wiki article on OSM is really bad by the way, it needs some investigation and revision. It's a very common American term, used in shop names, not just a UK term.

What we call a country store in North American English is more accurately tagged [shop=general](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dgeneral).  In the UK apparently they use the term "general store" for what we would call a farm store in the US, same as `shop=agrarian`.  For some reason in the UK there is a distinction, with `country_store` being for weathly hobby farmers, and `agrarian` being for commercial farmers I guess.  It doesn't really make any sense to me.  Here's another [discussion on the topic](https://github.com/openstreetmap/id-tagging-schema/pull/475#discussion_r886147323) for context.

**Comment by ZeLonewolf** [comment]
This is why we burned the Gaspee...

**Comment by bgo-eiu** [comment]
I should've learned about taking tags literally by now, I was sure that would be the tag for stores like this which usually have it in the name and are the only POI for a long distance https://www.openstreetmap.org/way/996972736#map=13/38.4577/-77.2446

Re: the topic though, airports, hospitals, universities and possibly sports stadiums cover quite a lot of what gets shown first in American maps. Schools seem tricky, there's just a lot of them in general

**Comment by 1ec5** [comment]
> I'm more inclined to think there's no such thing as a stereotypical American town than that the east coast is just a bit weird.

You’re right, I’m being a typical Midwesterner. 😅 I do think town/city halls would be the very likely anchor institutions where they exist, but this is not to suggest that we invent a town hall just to have it on the map. As for dollar stores, I think you’re underestimating the [ubiquity of Dollar General](https://www.npr.org/2017/12/11/569815331/loving-and-hating-dollar-general-in-rural-america) in the last couple decades. But I’m definitely not in favor of giving them more prominence than other retail POIs. If anything, they tend to be in the small towns that don’t have much other retail left, so there would be no need to give them special treatment.

Your point about country stores does remind me that, in some states, the local farm bureau would traditionally have been a community anchor institution, but I’m not sure there’s currently a way to distinguish them from any other insurance agency.

**Comment by kennykb** [comment]
" If anything, they tend to be in the small towns that don’t have much other retail left"  The other one, of course, is Wal-Mart.  :(

"Small towns typically don't have town halls" Around here, even the tiny ones do. They are often combined with the post office, the courthouse, the fire house, the police station, and the library. (Unless you're talking about towns that don't have home rule.)

"You’re right, I’m being a typical Midwesterner." Where a small town is usually built around a grain elevator, and the only institutions left are a church and a bar. (The retail shopping is at the Wal-Mart two towns over, and nowadays the kids are all bussed to the big school over in the county seat. Budgets, you know.)

I think that we're close to the right list of "things to show at smaller scale, without needing to include names."  

Beyond that, we're into maps that are themed for a particular consumer.  An old-school road map will show gas stations, restaurants and lodging, but likely only at the level of "which freeway exit accesses them".  A map for a long-distance trail will want to show grocery and convenience stores, coin laundries, lodging, outfitters, post offices, libraries, eateries - the services a hiker or touring cyclist will want on a town stop. A map put out by a commercial publisher will want to highlight advertisers. And so on.

**Comment by 1ec5** [comment]
> Greenspace and transportation facilities are important, but I think we can deal with them separately (and already have to some extent). Some of these are primarily landuse and only secondarily POIs, requiring different considerations. Maybe I shouldn’t have thrown cemeteries into the mix above, because you could make the same argument for the larger ones.

I removed cemeteries from the list above. We don’t have a good way to distinguish between a township’s general cemetery and other cemeteries anyways.

There’s a discussion in the context of #128 about the colors for each category of POI. The result of https://github.com/ZeLonewolf/openstreetmap-americana/issues/128#issuecomment-1159801218 is that the community anchor institutions would all be black or blue, unless we end up splitting out hospitals as red. Meanwhile, recreational POIs that we want to highlight at even lower zoom levels, such as campgrounds and rest areas (#443), would be either brown or green.

**Comment by kennykb** [comment]
I like blue for hospitals - the international white H on blue matches it.  Blue for police also makes sense in the US.  Red for fire stations, maybe.  And paint the bike shed Sherwin-WIlliams #7591 "Red Barn"
